plasma_framework/test/src/exits/payment/PaymentInFlightExitable.test.js

```diff
@@ -210,28 +212,7 @@ contract('PaymentInFlightExitable', ([_, alice, bob, carol]) => {
             });
         });
 
-        describe('when spending condition for output is not registered', () => {
```

Comment:
curious on the reason of removal of this test?

Reply:
This test was doubled in previous commit.